compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.kt

```diff
@@ -285,6 +285,11 @@ private fun Array<DataPointAtTime?>.set(index: Int, time: Long, dataPoint: Float
     }
 }
 
+/**
+ * Some platforms (e.g. iOS) velocity calculation is incorrect when end position is used in velocity calculation.
+ */
+internal expect val onlyPressedEventsForVelocityTracker: Boolean
```

Comment:
Suggestion: hide such details in ios-actual and use something like:
```kt
internal expect fun VelocityTracker.shouldUse(event: PointerInputChange)
```

Reply:
Sounds good.